pytket/pytket/circuit/display/__init__.py

```diff
@@ -73,3 +74,20 @@ def render_circuit_jupyter(
     :param circuit: the circuit to render.
     """
     render_circuit_as_html(circuit, True)
+
+
+def view_browser(circuit: Union[Dict[str, Union[str, float, dict]], Circuit]) -> None:
+    """Write circuit render html to a tempfile and open in browser.
+
+    Waits for some time for browser to load then deletes tempfile."""
+
+    fp = tempfile.NamedTemporaryFile(mode="w", suffix=".html", delete=False)
+    fp.write(cast(str, render_circuit_as_html(circuit)))
+    fp.flush()
+    fp.seek(0)
+    fp.close()
+    webbrowser.open(fp.name, new=2)
+
+    # give browser enough time to open before deleting file
+    time.sleep(5)
+    os.remove(fp.name)
```

Comment:
Can we use a `try` block to make this execute always even if an error is thrown above?